config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get '/projects/:id', to: "projects#show"
+  
+  get '/contestants', to: "contestants#index"
 end
```

Comment:
Your routes look great. You'll want to be consistent with if you're using double or single quotes though. Rails prefers double quotes as of Rails 7.